source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -3,16 +3,19 @@
 import * as AEL from "adaptive-expressions";
 
 class EvaluationContext {
-    private static readonly _reservedFields = ["$data", "$when", "$root", "$index"];
+    private static readonly _reservedFields = ["$data", "$when", "$root", "$index", "$_acTemplateVersion"];
```

Comment:
Need to update document somewhere for new _acTemplateVersion

Reply:
I created an issue to track updating documentation